environments/local-namespace.yml

```diff
@@ -2,6 +2,6 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    app: buyingcatalogue-dev
+    app: buyingcatalogue
```

Comment:
Are you sure this label should have changed?

Reply:
yes, this should be buyingcatalogue. It's a mistake on my part to add the -dev

Reply:
Had confirmation that -dev was to be removed